server/channels/api4/user.go

```diff
@@ -71,7 +71,7 @@ func (api *API) InitUser() {
 	api.BaseRoutes.UserByUsername.Handle("", api.APISessionRequired(getUserByUsername)).Methods("GET")
 	api.BaseRoutes.UserByEmail.Handle("", api.APISessionRequired(getUserByEmail)).Methods("GET")
 
-	api.BaseRoutes.User.Handle("/sessions", api.APISessionRequired(getSessions)).Methods("GET")
+	api.BaseRoutes.User.Handle("/sessions", api.RateLimitedHandler(api.APISessionRequired(getSessions), model.RateLimitSettings{PerSec: model.NewInt(2), MaxBurst: model.NewInt(10)})).Methods("GET")
```

Comment:
Yep, let's do whatever needs to be done.

Reply:
We don't hit this endpoint right now in the load-tests as far as I know, so if this is what we need to do to secure it, the load-test will have to comply, I'm not too worried about that.

But according to @jupenur's review, this may not be enough. If we limit the number of sessions per user on top of the limit on the user-agent length, then we wouldn't even need this rate limit, right?. That number of sessions per user could even be configurable, although it could also have a high enough hard-coded value, 0/5 on that, I don't know what use-cases our customers have regarding to this, to be honest.

An alternative would be to paginate this endpoint.